frontend/src/components/Status.tsx

```diff
@@ -54,117 +78,144 @@ function ClusterStatusIndicator({cluster}: {cluster: ClusterInfoSummary | Cluste
-    "COMPLETING": 'in-progress',
-    "CONFIGURING": 'loading',
-    "DEADLINE": 'info',
-    "FAILED": 'error',
-    "NODE_FAIL": 'error',
-    "OUT_OF_MEMORY": 'error',
-    "PENDING": 'pending',
-    "PREEMPTED": 'info',
-    "REQUEUED": 'info',
-    "REQUEUE_FED": 'info',
-    "REQUEUE_HOLD": 'info',
-    "RESIZING": 'info',
-    "RESV_DEL_HOLD": 'info',
-    "REVOKED": 'info',
-    "RUNNING": 'success',
-    "SIGNALING": 'info',
-    "SPECIAL_EXIT": 'info',
-    "STAGE_OUT": 'info',
-    "STOPPED": 'stopped',
-    "SUSPENDED": 'stopped',
-    "TIMEOUT": 'error',
+    BOOT_FAIL: 'error',
+    CANCELLED: 'error',
+    COMPLETED: 'success',
+    COMPLETING: 'in-progress',
+    CONFIGURING: 'loading',
+    DEADLINE: 'info',
+    FAILED: 'error',
+    NODE_FAIL: 'error',
+    OUT_OF_MEMORY: 'error',
+    PENDING: 'pending',
+    PREEMPTED: 'info',
+    REQUEUED: 'info',
+    REQUEUE_FED: 'info',
+    REQUEUE_HOLD: 'info',
+    RESIZING: 'info',
+    RESV_DEL_HOLD: 'info',
+    REVOKED: 'info',
+    RUNNING: 'success',
+    SIGNALING: 'info',
+    SPECIAL_EXIT: 'info',
+    STAGE_OUT: 'info',
+    STOPPED: 'stopped',
+    SUSPENDED: 'stopped',
+    TIMEOUT: 'error',
   };
 
-  return <StatusIndicator type={statusMap[status]}>
-    {status.replaceAll("_", " ")}
-  </StatusIndicator>
+  return (
+    <StatusIndicator type={statusMap[status]}>
+      {status.replaceAll('_', ' ')}
+    </StatusIndicator>
+  );
 }
 
-function ComputeFleetStatusIndicator({status}: {status: ComputeFleetStatus})
-{
+function ComputeFleetStatusIndicator({status}: {status: ComputeFleetStatus}) {
   const statusMap: Record<ComputeFleetStatus, StatusIndicatorProps.Type> = {
-    "START_REQUESTED": 'loading',
-    "STARTING": 'pending',
-    "RUNNING": 'success',
-    "PROTECTED": 'stopped',
-    "STOP_REQUESTED": 'loading',
-    "STOPPING": 'stopped',
-    "STOPPED": 'stopped',
-    "UNKNOWN": 'info',
-    "ENABLED": 'success',
-    "DISABLED": 'stopped',
+    START_REQUESTED: 'loading',
+    STARTING: 'pending',
+    RUNNING: 'success',
+    PROTECTED: 'stopped',
+    STOP_REQUESTED: 'loading',
+    STOPPING: 'stopped',
+    STOPPED: 'stopped',
+    UNKNOWN: 'info',
+    ENABLED: 'success',
+    DISABLED: 'stopped',
   };
 
-  return <StatusIndicator type={statusMap[status]}>
-    {status.replaceAll("_", " ")}
-  </StatusIndicator>
+  return (
+    <StatusIndicator type={statusMap[status]}>
+      {status.replaceAll('_', ' ')}
+    </StatusIndicator>
+  );
 }
 
-function InstanceStatusIndicator({instance}: {instance: EC2Instance | Instance})
-{
+function InstanceStatusIndicator({
+  instance,
+}: {
+  instance: EC2Instance | Instance;
+}) {
   const statusMap: Record<InstanceState, StatusIndicatorProps.Type> = {
-    "pending": 'pending',
-    "running": 'success',
-    "shutting-down": 'loading',
-    "stopped": 'stopped',
-    "stopping": 'pending',
-    "terminated": 'stopped',
+    pending: 'pending',
+    running: 'success',
+    'shutting-down': 'loading',
```

Comment:
question(non-blocking): Why this needs quotes?

Reply:
because it wouldn't be valid syntax otherwise 

```ts
const obj = {
  a-b: "anything" // syntax error
}
```

Reply:
ok, at that point I'd put all the variables inside some single or double quotes to avoid possible mistakes, but apparently the linter does not agree with me :D